analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/DisablingRequestValidationBase.cs

```diff
@@ -47,7 +47,7 @@ protected override void Initialize(SonarAnalysisContext context)
                 SymbolKind.NamedType,
                 SymbolKind.Method);
 
```

Comment:
`DiagnosticId` can be made private

Reply:
I tried to avoid nonrelated changes as much as I could (didn't always succeeded)